templates/data/relational-databases.html

```diff
@@ -0,0 +1,342 @@
+{% extends 'base_index.html' %}
+
+{% block title %}Relational databases within Canonical{% endblock %}
```

Comment:
```suggestion
{% block title %}What is a relational database?{% endblock %}
```

Reply:
This suggestion doesn't match [copy doc](https://docs.google.com/document/d/1ikYzd23-zop6k7qO7LXzjttvltBL-au8A50XzuDDd5M/edit#heading=h.ic5iz9wp2hp)?

Reply:
It does, https://docs.google.com/document/d/1ikYzd23-zop6k7qO7LXzjttvltBL-au8A50XzuDDd5M/edit?disco=AAAA8JqqpvU

Reply:
Ah I see, it's defined in 2 places in the same doc. As this still needs UX review, tagging @juanruitina so he can take a look at this when he's back on Monday